docs/book/src/using/cli.md

```diff
@@ -29,7 +29,7 @@ Analyzes a repository or single pull request with a set of analysis tools. Hipch
 
 A github token must be set in the HC_GITHUB_TOKEN system environment variable for `hc check` to run. Hipcheck only needs permissions for accessing public repository data, so those are the only permissions to assign to your generated token.
 
-There are currently two working secondary subcommands under `check`. Running `hc check` without a subtask runs `hc help check`. The available subcommands are:
+There are currently six working secondary subcommands under `check`. Running `hc check` without a subtask runs `hc help check`. The available subcommands are:
```

Comment:
I don't think the SPDX check is working yet either. Which puts us at 5 working subcommands.

Reply:
Should `SPDX` be removed, or distinguished as a yet-unfinished command but left in the book?

Reply:
I would remove it from the book for now. We even took the preliminary code to handle it out of the repo because it wasn't finished yet. When I update the CLI, I will indicate that its a future feature there.